connectors/cli/index.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+
+from elasticsearch import ApiError
+
+from connectors.es import ESClient
+
+
+class Index:
+    def __init__(self, config):
+        self.elastic_config = config
+        self.es_client = ESClient(self.elastic_config)
+
+    def list_indices(self):
+        return asyncio.run(self.__list_indices())["indices"]
+
+    def clean(self, index_name):
+        return asyncio.run(self.__clean_index(index_name))
+
+    def delete(self, index_name):
+        return asyncio.run(self.__delete_index(index_name)) is None
```

Comment:
I assume because [this](https://github.com/elastic/connectors/blob/main/connectors/es/client.py#L176) method returns None if there are no exceptions.